src/app/containers/Ad/Amp/index.jsx

```diff
@@ -112,29 +105,31 @@ const AmpAd = ({ slotType }) => {
       aria-label={ariaLabel}
       role="region"
       data-e2e="advertisement"
-      amp-access="toggles.ads.enabled"
-      amp-access-hide="true"
       aria-hidden="true"
     >
-      <StyledWrapper>
-        <Helmet>
-          {AMP_ADS_JS}
-          {AMP_ACCESS_JS}
-          {AMP_ACCESS_FETCH(service)}
-        </Helmet>
-
-        <StyledAd>
-          <StyledLink
-            href={LABEL_LINK}
-            script={script}
-            service={service}
-            dir={dir}
+      <Helmet>
```

Comment:
Comparing this container with the `Canonical` one, I'm wondering if we can move the `Helmet` bit above the `AdSection` for consistency.

Reply:
Thanks @DenisHdz . This has been done to match the structure of the Canonical